src/components/RootNavigator/BaseNavigator.js

```diff
@@ -156,6 +157,16 @@ export default function BaseNavigator() {
           headerRight: null,
         }}
       />
+      <Stack.Screen
+        name={ScreenName.SwapV2FormSelectFees}
+        component={SwapFormV2SelectFees}
+        options={{
+          headerTitle: () => (
+            <StepHeader title={t("transfer.swap.form.summary.fees")} />
```

Comment:
I haven't followed the rework of mobile as much as desktop but we need to be completely detached from the components and wording of the previous version, and be able to toggle between them. Wording should be namespaced to v2 like on lld